docs/getting-started/server/ef/index.md

```diff
@@ -6,27 +6,36 @@ Entity Framework support is still in beta and is not suitable for production dat
 
 :::
 
+:::info
```

Comment:
two didn't seem like too many to me, but really the info isn't necessary. It is _always_ the case that this page is for development, not production.

It just felt relevant since the `deployment` page was previously the place to go for EF beta deployments.

Reply:
what's wrong with callouts?

Reply:
True, probably just me that dislike callouts below each other.